If someone knows what is supposed to be failing here, I'm having a hard time understanding the logs if `test_sphinx.sh`

There are many errors that look like this, each for a different pdf, svg file:
```
2023-01-17T11:33:47.3494530Z ERROR: chromium --headless --disable-gpu --disable-software-rasterizer --print-to-pdf=/home/runner/work/sympy/sympy/doc/_build/doctrees-latex/images/kin_2pt.pdf /home/runner/work/sympy/sympy/doc/src/modules/physics/vector/kin_2pt.svg.html
2023-01-17T11:33:47.5274352Z [0117/113347.527269:WARNING:bluez_dbus_manager.cc(247)] Floss manager not present, cannot set Floss enable/disable.
2023-01-17T11:33:47.5286304Z [0117/113347.528389:ERROR:gpu_init.cc(523)] Passthrough is not supported, GL is disabled, ANGLE is 
2023-01-17T11:33:47.6294959Z [0117/113347.629323:INFO:headless_shell.cc(223)] 18708 bytes written to file /home/runner/work/sympy/sympy/doc/_build/doctrees-latex/images/kin_2pt.pdf
2023-01-17T11:33:47.7906958Z Chromium 109.0.5414.74 snap
```
Then there's this:
```
2023-01-17T11:34:26.2329300Z Build finished; the LaTeX files are in _build/latex.
2023-01-17T11:34:26.2335684Z Set the environment variable LATEXMKOPTS='-xelatex -silent'
2023-01-17T11:34:26.2341625Z And run `make all' in that directory to run these through xelatex.
2023-01-17T11:34:26.2343332Z make -C _build/latex
2023-01-17T11:34:26.2363811Z make[1]: Entering directory '/home/runner/work/sympy/sympy/doc/_build/latex'
2023-01-17T11:34:26.2364525Z latexmk -pdf -dvi- -ps- -halt-on-error -xelatex -silent 'sympy-1.12.dev.tex'
2023-01-17T11:34:26.3442896Z Latexmk: Run number 1 of rule 'xelatex'
2023-01-17T11:34:26.3984985Z This is XeTeX, Version 3.14159265-2.6-0.999991 (TeX Live 2019/Debian) (preloaded format=xelatex)
2023-01-17T11:34:26.4037074Z  restricted \write18 enabled.
2023-01-17T11:34:26.5226817Z entering extended mode
2023-01-17T11:34:39.8960331Z Latexmk: Summary of warnings from last run of (pdf)latex:
2023-01-17T11:34:39.8961386Z   Latex failed to resolve 3349 reference(s)
2023-01-17T11:34:39.8961881Z   =====Latex reported missing or unavailable character(s).
2023-01-17T11:34:39.8962257Z =====See log file for details.
2023-01-17T11:34:39.8962814Z   Latex failed to resolve 701 citation(s)
2023-01-17T11:34:39.8963250Z Collected error summary (may duplicate other messages):
2023-01-17T11:34:39.8963930Z   xelatex: Command for 'xelatex' gave return code 1
2023-01-17T11:34:39.8964361Z       Refer to 'sympy-1.12.dev.log' for details
2023-01-17T11:34:39.8964799Z Latexmk: Use the -f option to force complete processing,
2023-01-17T11:34:39.8965235Z  unless error was exceeding maximum runs, or warnings treated as errors.
2023-01-17T11:34:39.8987465Z make[1]: *** [Makefile:29: sympy-1.12.dev.pdf] Error 12
2023-01-17T11:34:39.8988499Z make: *** [Makefile:116: latexpdf] Error 2
2023-01-17T11:34:39.8990193Z make[1]: Leaving directory '/home/runner/work/sympy/sympy/doc/_build/latex'
2023-01-17T11:34:39.8990761Z An error had occurred during the LaTeX build
```
Then a lot of reference errors like this:
```
2023-01-17T11:34:39.9204793Z LaTeX Warning: Reference `modules/matrices/matrices:sympy.matrices.dense.hessia
2023-01-17T11:34:39.9205176Z n' on page 1353 undefined on input line 110148.
```
And it finishes like this:
```
2023-01-17T11:34:39.9205326Z 
2023-01-17T11:34:39.9205387Z [1353] [1354]
2023-01-17T11:34:39.9205578Z ! Misplaced \omit.
2023-01-17T11:34:39.9205819Z \math@cr@@@ ...@ \@ne \add@amps \maxfields@ \omit 
2023-01-17T11:34:39.9206183Z                                                   \kern -\alignsep@ \iftag@ ...
2023-01-17T11:34:39.9206436Z l.110337 \end{align}\end{split}
2023-01-17T11:34:39.9206648Z                                 
2023-01-17T11:34:39.9206944Z Here is how much of TeX's memory you used:
2023-01-17T11:34:39.9207172Z  17934 strings out of 481804
2023-01-17T11:34:39.9207406Z  377219 string characters out of 5937876
2023-01-17T11:34:39.9207646Z  756530 words of memory out of 5000000
2023-01-17T11:34:39.9207906Z  34488 multiletter control sequences out of 15000+600000
2023-01-17T11:34:39.9208200Z  548661 words of font info for 146 fonts, out of 8000000 for 9000
2023-01-17T11:34:39.9208531Z  15 hyphenation exceptions out of 8191
2023-01-17T11:34:39.9208815Z  43i,26n,67p,6438b,765s stack positions out of 5000i,500n,10000p,200000b,80000s
2023-01-17T11:34:39.9208993Z 
2023-01-17T11:34:39.9209217Z Output written on sympy-1.12.dev.xdv (1358 pages, 25585032 bytes).
2023-01-17T11:34:40.9084172Z ##[error]Process completed with exit code 1.
2023-01-17T11:34:40.9147759Z Post job cleanup.
2023-01-17T11:34:41.0490835Z [command]/usr/bin/git version
2023-01-17T11:34:41.0549063Z git version 2.39.0
2023-01-17T11:34:41.0604784Z Temporarily overriding HOME='/home/runner/work/_temp/6c114a08-b064-4d85-af5a-0878282b8a50' before making global git config changes
2023-01-17T11:34:41.0606013Z Adding repository directory to the temporary git global config as a safe directory
2023-01-17T11:34:41.0611914Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/sympy/sympy
2023-01-17T11:34:41.0656718Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2023-01-17T11:34:41.0696307Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2023-01-17T11:34:41.0969899Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2023-01-17T11:34:41.0990293Z http.https://github.com/.extraheader
2023-01-17T11:34:41.1009955Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2023-01-17T11:34:41.1058107Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2023-01-17T11:34:41.1553995Z Cleaning up orphan processes
```

--- a/sympy/matrices/dense.py
+++ b/sympy/matrices/dense.py
@@ -207,7 +207,7 @@ def rot_givens(i, j, theta, dim=3):
         \begin{bmatrix}
             g_{jj} & g_{ji} \\
             g_{ij} & g_{ii}
-        \end{bmatrix} = 
+        \end{bmatrix} =
         \begin{bmatrix}
             c & -s \\
             s & c

--- a/sympy/matrices/dense.py
+++ b/sympy/matrices/dense.py
@@ -197,13 +197,21 @@ def rot_givens(i, j, theta, dim=3):
 
     For fixed ``i > j``\, the non-zero elements of a Givens matrix are
     given by:
+        - $g_{kk} = 1$ for $k \ne i,\,j$
+        - $g_{kk} = c$ for $k = i,\,j$
+        - $g_{ji} = -g_{ij} = -s$
+
+    Which gives the following rotation submatrix:
 
     .. math::
-        \begin{align}
-            g_{kk} &= 1 \qquad \text{for} \ k \ne i,\,j \\
-            g_{kk} &= c \qquad \text{for} \ k = i,\,j \\
-            g_{ji} &= -g_{ij} = -s
-        \end{align}
+        \begin{bmatrix}
+            g_{jj} & g_{ji} \\
+            g_{ij} & g_{ii}
+        \end{bmatrix} = 
+        \begin{bmatrix}
+            c & -s \\
+            s & c
+        \end{bmatrix}
 
     Parameters
     ==========
@@ -272,6 +280,10 @@ def rot_givens(i, j, theta, dim=3):
         raise ValueError('dim must be an integer biggen than one, '
                          'got {}.'.format(dim))
 
+    if i == j:
+        raise ValueError('i and j must be different, '
+                         'got ({}, {})'.format(i, j))
+
     for ij in [i, j]:
         if not isinstance(ij, int) or ij < 1 or ij > dim:
             raise ValueError('i and j must be between 1 and '
@@ -281,7 +293,6 @@ def rot_givens(i, j, theta, dim=3):
     c = cos(theta)
     s = sin(theta)
     M = eye(dim)
-
     M[i-1, i-1] = c
     M[j-1, j-1] = c
     M[i-1, j-1] = s

--- a/sympy/matrices/dense.py
+++ b/sympy/matrices/dense.py
@@ -208,10 +208,10 @@ def rot_givens(i, j, theta, dim=3):
             g_{jj} & g_{ji} \\
             g_{ij} & g_{ii}
         \end{bmatrix} =
-        \begin{bmatrix}
-            c & -s \\
-            s & c
-        \end{bmatrix}
+            \begin{bmatrix}
+                c & -s \\
+                s & c
+            \end{bmatrix}
 
     Parameters
     ==========

